otherlibs/unix/unix.mli

```diff
@@ -227,6 +227,14 @@ val execv : string -> string array -> 'a
    Using {!create_process} or one of the [open_process_*] functions
    instead is recommended.
 
+   The first element of the array [args] will often be [argv[0]] or
```

Comment:
Using `will often be` doesn't really make sense for a specification, you are not documenting an optional behavior: programs will break if they are not send the correct number of arguments.

Reply:
An alternative wording might be:

> On both unix and windows, the convention is that the first element of the array [args] is the name of the executed program.